src/shared/report_op.c

```diff
@@ -472,13 +474,30 @@ void os_ReportdStart(report_filter *r_filter)
         }
 
         alerts_filtered++;
-        data_to_clean = (alert_data **) os_AddPtArray(al_data, (void **)data_to_clean);
+
+        /* If we have to dump the alerts */
+        if (r_filter->show_alerts) {
+            if (!msg_show_alerts) {
+                os_calloc(size_max_msg, sizeof(char), msg_show_alerts);
+                size_used_msg = snprintf(msg_show_alerts, size_max_msg, "Log dump:\n------------------------------------------------\n");
```

Comment:
Why that message of log dump? could you share a report example? Thanks.

Reply:
The message of `log dump`, is the same that we use right now, the only thing I changed is the preparation of the message.